XPathParser/project.json

```diff
@@ -11,10 +11,11 @@
     "repository": {
       "type": "git",
       "url": "https://github.com/quamotion/XPathParser"
-    },
+    }
   },
 
   "dependencies": {
+    "NETStandard.Library": "1.6.0"
```

Comment:
To be honest, this is the first time I've really started messing around with project.json files. So basically that seemed to be enough to make things compile correctly. I can see if it can be done another way.